src/index.css

```
@@ -53,181 +85,589 @@
 
   body {
     @apply bg-background text-foreground;
-    background: #1e1e2e;
-    background-image: none;
+    background: radial-gradient(circle at top, var(--c-bg-top), var(--c-bg) 60%, var(--c-bg-bottom));
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)